docs.js

```diff
@@ -0,0 +1,127 @@
+const blessed = require('blessed')
+const contrib = require('blessed-contrib')
+const walkup = require('walk-up')
+const fs = require('fs').promises
+const { join } = require('path')
+
+function findDocs () {
+  return new Promise((resolve, reject) => {
+    const searchDir = 'node_modules/fastify/docs'
```

Comment:
You might want to use `require.resolve` instead. Yarn pnp and pnpm would not find this otherwise.

Reply:
Can require.resolve be used for resolving a directory? I thought it could only resolve .js files?

Reply:
you can resolve the entry point and then move in that folded

Reply:
Fair, I'll do this! Ty